guides/cookbook.md

```diff
@@ -31,141 +80,177 @@ constructing input streams to read files), which is out of scope for this
 cookbook. Code such as this will be replaced by a method with an empty (but
 implied) implementation.
 
-> *Java*
-> 
-> Import statements for classes internal to the Ion library or to `java.lang`
```

Comment:
Was this removed for not being useful? If it's still useful, we can retain it and have tabs here for any language-specific cookbook usage info.

Reply:
removed for not being useful